ci-operator/step-registry/openshift/manifests/external/openshift-manifests-external-commands.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+LOCALPATH="${SHARED_DIR}/manifest_external.yaml"
+echo "${SHA256_HASH} -" > /tmp/sum.txt
+if ! curl -fLs "${URL}" | tee "${LOCALPATH}" | sha256sum -c /tmp/sum.txt >/dev/null 2>/dev/null; then
+  echo "Expected file at ${URL} to have checksum ${SHA256_HASH} but instead got $(< "${LOCALPATH}" sha256sum | cut -d' ' -f1)"
+  exit 1
+fi
+echo "Downloaded ${URL}, sha256 checksum matches ${SHA256_HASH}"
+
+# Check file syntax
+pip3 install pyyaml --user
+python3 -c 'import yaml
+import sys
+data = yaml.safe_load_all(open(sys.argv[1]))' "${LOCALPATH}"
+
+if [ "${POST_INSTALL}" == "true" ]; then
+  oc apply -f "${LOCALPATH}"
```

Comment:
and then remove `${LOCALPATH}` on success?  No need to add content under `SHARED_DIR` once we've successfully pushed it into the cluster.

Reply:
Done in a9207c030fed082e30a639e4db52a47edefa1e66